testsuite/integration/clustering/pom.xml

```diff
@@ -810,8 +810,8 @@
                                 <configuration>
                                     <!-- Tests to execute. -->
                                     <includes>
-                                        <include>org/jboss/as/test/clustering/cluster/jpa2lc/*TestCase.java</include>
-                                        <include>org/jboss/as/test/clustering/cluster/jpa2lcCustomRegion/*TestCase.java</include>
+                                        <include>org/jboss/as/test/clustering/cluster/jpa/*TestCase.java</include>
+                                        <include>org/jboss/as/test/clustering/cluster/jpa/custom/*TestCase.java</include>
```

Comment:
I think you meant this to be an addition of more tests instead of a replacement.

Reply:
Ok; good. In e026da92aaf9fb5e4f36e123674c5b8ff89405f5 Paul changed the package.